### PR TITLE
Remove dependence on newman image in code builds

### PR DESCRIPTION
**Pull request by jgondron**
This is to avoid using a custom image for our CodeBuilds. I was able to
move Beehive pipeline to using the ndlib-cdk provided NewmanRunner, but
could not do this for things like Buzz. Buzz requires an extra
`--ignore-redirects` and we don't have a way to extend the NewmanRunner
to do that yet, so just converted the test CodeBuild in the RailsPipeline
to install at runtime similar to the NewmanRunner.

### CODE REVIEW

File: src/pipeline-constructs/rails-pipeline.ts [hunk 1]
```diff
@@ -279,6 +278,16 @@ export class RailsPipeline extends Construct {
     const smokeTestsProject = new PipelineProject(this, `${props.namespace}-SmokeTests`, {
```
Comment (mlkirkpatrick) [inline src/pipeline-constructs/rails-pipeline.ts:278]: Could use `NewmanRunner` from ndlib-cdk, I think?

**Review by mlkirkpatrick — APPROVED** [review]

File: src/pipeline-constructs/rails-pipeline.ts (hunk 1 above)
Comment (jgondron) [reply src/pipeline-constructs/rails-pipeline.ts:278]: Nah, this is something I mentioned in the PR desc. I need to add an `--ignore-redirects` switch for some of these. I didn't see a way to add additional cli switches to the NewmanRunner. Something we should probably add

Comment (mlkirkpatrick) [reply src/pipeline-constructs/rails-pipeline.ts:278]: Ah yes, I missed that, sorry. 😄

**Comment by jgondron** [comment]
> So, did you just recently add NewmanRunner to ndlib-cdk? Or are you just taking advantage of something done a while ago?

More the latter. Some of these pipelines were created prior to it's inclusion, so we're able to take advantage of it now.  Some of these I wrote and just forgot we had added it to ndlib-cdk.

**Pull request merged**